trieste/models/interfaces.py

```diff
@@ -86,6 +86,7 @@ def predict_y(self, query_points: TensorType) -> tuple[TensorType, TensorType]:
             ``query_points``. For a predictive distribution with event shape E, the mean and
             variance will both have shape [...] + E.
         """
+        pass  # (required so that mypy doesn't think this method is abstract)
```

Comment:
aaah, I was getting these errors - I was wondering whats going on with mypy, nice that this resolves it!